session1/solutions/assignment4/Rakefile

```diff
@@ -1 +1,42 @@
 # assignment 4 solution
+ #namespace :gziped_files do
+  
+  desc "Archive files to gzip to a specified path"
+  task :archive , [:M_PATH, :EXCLUDE]  do
+
+  mpath = ENV["M_PATH"]
+  dexclude = ENV["EXCLUDE"]
+
+    # Find all of the files 
+    Dir.chdir mpath
```

Comment:
There is no need to change dir's - any ruby/rake command can get either relative or full paths

Reply:
When i deleted the changing dir command, it compress the all folders in the path